### PR TITLE
Fix some typos in codebase

### PR DESCRIPTION
**Pull request by Evangelink**
## Description

Fix some typos in codebase.

### CODE REVIEW

File: src/Microsoft.TestPlatform.AdapterUtilities/Resources/xlf/Resources.xlf [hunk 1]
```diff
@@ -36,12 +36,12 @@ Example: 'System.Reflection.MethodBase' is not implemented on this platform!</no
       <trans-unit id="ErrorInvalidSequenceAt">
```
Comment (Evangelink) [inline src/Microsoft.TestPlatform.AdapterUtilities/Resources/xlf/Resources.xlf:36]: I did a rebuild but I can't get the other xlf files to be updated, is there any specific command to run?

Comment (nohwnd) [reply src/Microsoft.TestPlatform.AdapterUtilities/Resources/xlf/Resources.xlf:36]: Just build.cmd should do it.

Comment (nohwnd) [reply src/Microsoft.TestPlatform.AdapterUtilities/Resources/xlf/Resources.xlf:36]: @Haplois  I've built this locally as well and I don't get updates in the localization files either. Any tips?

Comment (Haplois) [reply src/Microsoft.TestPlatform.AdapterUtilities/Resources/xlf/Resources.xlf:36]: We only update main resource file and the xlf file. All others are updated by the bots.

**Review by Haplois — APPROVED** [review]

**Pull request merged**